src/bouwdossiers/batch.py

```diff
@@ -140,7 +165,13 @@ def add_wabo_dossier(x_dossier, file_path, import_file, count, total_count):  #
         activiteiten=activiteiten
     )
 
-    bouwdossier.save()
+    try:
+        with transaction.atomic():
+            bouwdossier.save()
+    except IntegrityError as e:
+        log.error(f"Exception while saving {dossier} in {file_path} with : {e}")
+        return count, total_count
+
```

Comment:
@kramer65 dit is significant anders dan voorheen. Is dat voor onze projecten een issue?

Reply:
Het verschil is dat wanneer er bij import van een record in een XML een fout optreedt door bijv duplicate key, dit nietde verwerking andere records na dit record verhinderd. Dit maakt het,mogelijk bij import voor een XML file om alle duplicate keys of andere fouten in de XML file te zien